src/pytezos/operation/result.py

```diff
@@ -59,7 +64,7 @@ def consumed_gas(operation_group: Dict[str, Any]) -> int:
         """
         return sum(
             map(
-                lambda x: int(x.get('consumed_gas', '0')),
+                lambda x: math.ceil(int(x.get('consumed_milligas', '0')) / 1000),
```

Comment:
This is my understanding why it complained gas limit too low, consumed_gas has been removed from kathmandu. 

see
https://app.datadoghq.com/logs?query=host%3A%22arn%3Aaws%3Alambda%3Aus-east-1%3A950473261671%3Afunction%3Atezos-tokens-staging-ownership_processor%22%20service%3Atezos-tokens-staging-ownership_processor%20%40aws.awslogs.logGroup%3A%22%2Faws%2Flambda%2Ftezos-tokens-staging-ownership_processor%22&cols=host%2Cservice&context_event=AYNMM1LNAABzO0bPWb3OpwAC&event=AQAAAYNMMpWp66-DewAAAABBWU5NTXJseUFBRHFIRU01UV9LU0lnQUE&index=&messageDisplay=inline&stream_sort=time%2Cdesc&to_event=AQAAAYNMNEF7mZu-egAAAABBWU5NTkYtNUFBRHRyZDFoM0FwX2p3QU8&viz=&from_ts=1663427940085&to_ts=1663430836603&live=false